driver/pgdriver/config.go

```diff
@@ -283,6 +287,10 @@ func parseDSN(dsn string) ([]Option, error) {
 			// (verify chain, but skip server name).
 			// See https://github.com/golang/go/issues/21971 .
```

Comment:
Please update the comment and explain why this is required

Reply:
The comment is still correct. `InsecureSkipVerify = true` will not verify either the certificate chain nor verify the server name. The `ServerName` field only adds the server name to the TLS ClientHello but doesn't impact the verification if insecure is set.

Reply:
Good know, thanks.